client/worker.go

```diff
@@ -28,10 +28,10 @@ import (
 
 	"github.com/golang/protobuf/proto"
 	jaegerpb "github.com/jaegertracing/jaeger/model"
-	ocTrancelator "github.com/open-telemetry/opentelemetry-collector/translator/trace"
 	"go.opencensus.io/trace"
 
 	sapmpb "github.com/signalfx/sapm-proto/gen"
+	"github.com/signalfx/sapm-proto/translator"
```

Comment:
Made this change to get rid of opentelemetry-collector dependency which pulled in a huge tree of it's own deps that eventually caused a lot of pain signalfx-agent (through signalfx/golib). I think copying a bit of code is anyway better than having a big dependency but in this case it's really required in order to not have very ugly go modules hacks in signalfx-agent.

Reply:
Which makes me think that we should be able to import things from the opentelemetry-collector infrastructure without having to bring dependencies related to specific receivers/exporters. Not sure if it would have fixed this issue but reduces the surface/dependencies anyway.